keys/90263F5E34B5D26F841B2D70B905A377C78E929C.asc

```diff
@@ -0,0 +1,11 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+xjMEYPuc/RYJKwYBBAHaRw8BAQdA2f29j7PbuAHzJ/qce4wHGxOa/kY+JhYqk972GJZkvPPNFURl

```

Comment:
Please add the key info here as well.

Reply:
fix: https://github.com/GNUWeeb/pgpkeys/pull/6/commits/2a41258be7f71d5fe07d7d919c81a54ffa22d3db